.ci/aws/common.groovy

```diff
@@ -69,6 +75,13 @@ def run_test_orchestrator_once(run_name, build_tag, os, instance_type, instance_
     kill_all_clusters(instance_type, region)
     wait_for_odcr_capacity(region, instance_count, odcr)
 
+    /*
+     * p3dn clusters are getting ICE'ed within an ODCR, when we try to launch them back to back.
+     * This is a non-deterministic work around to help us increase our chances of not getting ICE'ed.
+     * Worst case, this increases our time to publish results on PR's by 15 minutes.
+     */
+    p3dn_sleep_5min(instance_type)
```

Comment:
why have a function here?  Put the code and the comment in the same place.

Reply:
done